packages/@lwc/engine-core/src/framework/profiler.ts

```diff
@@ -120,7 +127,7 @@ export function logOperationStart(opId: OperationId, vm: VM) {
     }
 
     if (isProfilerEnabled) {
-        currentDispatcher(opId, Phase.Start, vm.tagName, vm.idx);
+        currentDispatcher(opId, Phase.Start, vm.tagName, vm.idx, vm.renderMode, vm.shadowMode);
```

Comment:
Adding two additional arguments here makes this change backwards-compatible. Existing code using the profiler won't be looking for these two arguments.